rainier-core/src/main/scala/com/stripe/rainier/ir/Tracer.scala

```diff
@@ -97,6 +97,7 @@ object Tracer {
             s"if($t == 0.0) $z else $nz"
           else
             s"(if($t == 0.0) $z else $nz)"
+        case s: SeqIR => ???
```

Comment:
This goes away with the new tracer, right?

Reply:
correct, which is why I left it this way :)

Reply:
I'll do the rebase.